service/src/main/java/zone/cogni/asquare/service/rest/model/RestResponse.java

```diff
@@ -44,10 +44,7 @@ public String getTitle() {
   }
 
   public String getCode() {
-    if (StringUtils.isEmpty(code)) {
-      return "";
-    }
-    return code;
+    return StringUtils.hasText(code) ? code : "";
```

Comment:
not sure if we should enforce usage of apache commons lang3 if needed function is available?